### ISSUE TITLE
Help Matrice 100 Raspberry Pi 3 b Ros Setup

### ISSUE THREAD

**Issue by Matthewsw1234** [dji-sdk/Onboard-SDK#900]
Hi everyone. I know the Matrice 100 is a dated platform by now but I was wanting to develope some software for it. Could anyone give me a new guide to connecting the drone to a raspberry pi and do a walkthrough of the whole setup. Right now I just have a pi with no software on it besides the Rasp Os. Honestly i'm new to prgramming and know next to nothing about what I am doing. If someone could either guide me from step one to the end on how to do this it would be great. Once everything is connected and working with simple commands like "takeoff" "landing" etc it would be great. I would be ok but all current guides for this are either dated and don't work or are running different software. 

Thanks!

**Comment by dji-dev** [comment]
Agent comment from kyle.cai in <a href='https://djisdksupport.zendesk.com/agent/tickets/70965'>Zendesk ticket #70965</a>:<br><br>M100 is recommended to use OSDK 3.9 version. If you need to use ROS, OSDK linux 3.9 + OSDK ROS 3.8 is recommended. The offline documentation can be downloaded from the developer website. There are tutorials on hardware connection and SDK sample compilation.
https://developer.dji.com/document/1be2b992-5862-4eb5-819b-dd6036f76133
 ![](https://djisdksupport.zendesk.com/attachments/token/hbZuo1Z21T2xJZpKdGKL4K2fr/?name=image.png)<br><a href='https://djisdksupport.zendesk.com/attachments/token/hbZuo1Z21T2xJZpKdGKL4K2fr' target='zd_attach'>image.png</a>
<br><p><em><small>°°°</small></em></p>

**Comment by Matthewsw1234** [comment]
> Agent comment from kyle.cai in [Zendesk ticket #70965](https://djisdksupport.zendesk.com/agent/tickets/70965):M100 is recommended to use OSDK 3.9 version. If you need to use ROS, OSDK linux 3.9 + OSDK ROS 3.8 is recommended. The offline documentation can be downloaded from the developer website. There are tutorials on hardware connection and SDK sample compilation. https://developer.dji.com/document/1be2b992-5862-4eb5-819b-dd6036f76133 ![](https://camo.githubusercontent.com/179b883c32ebe2b038df6e0d6bb1ab8b30ae5e782e235e578afb6e460f5aadfe/68747470733a2f2f646a6973646b737570706f72742e7a656e6465736b2e636f6d2f6174746163686d656e74732f746f6b656e2f68625a756f315a32315432784a5a704b64474b4c344b3266722f3f6e616d653d696d6167652e706e67)[image.png](https://djisdksupport.zendesk.com/attachments/token/hbZuo1Z21T2xJZpKdGKL4K2fr)
> 
> _°°°_

Hi. Thanks for responding. I see that but it is hard to follow as most of the links and pictures don’t work anymore. What I’m mainly trying to figure out is if I’m using a Raspberry Pi3 B, what OS and software do I use?
